src/main/java/project/ProjectFinal.java

```
@@ -0,0 +1,233 @@
+        System.out.println("----------------------------------");
+    }
+    
+    static void logon(){
+        System.out.println("Please enter your username");
+        Scanner sc = new Scanner(System.in);
+        String username = sc.nextLine();
+        System.out.println("Please enter your password");
+        Scanner sc2 = new Scanner(System.in);
+        String password = sc2.nextLine();
+        
+        if (username.equals("user") && password.equals("password")){
+        User = true;
+        System.out.println("user logged in");
+        menu();
+        optionSelect();
+    }
+        else if (username.equals("admin") && password.equals("password")){
+            Admin = true;
+            System.out.println("admin logged in");
+            menu();
+            optionSelect();
+        }
+        else{
+            System.out.println("logon failed, please try again");
+            menu();
+            optionSelect();
+        }
+        
+        
+    }
+    
+    public static void addData(){
+        
+        System.out.println("Please enter your full name");
+        Scanner sc = new Scanner(System.in);
+        String name = sc.nextLine();
+        Scanner sc2 = new Scanner(System.in);
+        System.out.println("Please enter your contact number");
+        String number = sc.nextLine();
+        Scanner sc3 = new Scanner(System.in);
+        System.out.println("Please enter your full address");
+        String address = sc.nextLine();
+        
+        try {
+            try (FileWriter myWriter = new FileWriter("C:\\git\\pkgfinal\\project.txt")) {
+                myWriter.write(name + "\t" + number + "\t" + address);
+                myWriter.write("\n");
+                myWriter.flush();
+                myWriter.close();
+                menu();
+                optionSelect();
+            }
+        } catch (IOException ex) {
+            System.out.println("Error occured, please try again" + ex.getStackTrace());
```

You are swallowing your exception, consider showing it to the user?

Or log the error to a file.